amgut/db/patches/0018.sql

```diff
@@ -0,0 +1,65 @@
+-- August 19, 2015
```

Comment:
yes, tested on production copy of the database and we lose 6 improperly formatted times, rest are converted fine. We lose no dates.